src/services/dashboards/dashboards.svc.coffee

```diff
@@ -110,26 +110,47 @@ angular
             (success)->
               orgId = success.currentOrganization.id
 
+              # Retrieve dashboards
               $http.get(ImpacRoutes.dashboards.index(orgId)).then(
                 (dashboards)->
                   _self.setDashboards(dashboards.data).then(->
                     _self.setCurrentDashboard()
                     deferred.resolve(_self.config)
                     $log.info("Impac! - DashboardsSvc: loaded (force=#{force})")
```

Comment:
DashboardSvc may not really be loaded here. This could log as successful, but the widget retrieval below could fail.